front/app/hooks/useExceedsSeats.ts

```diff
@@ -1,6 +1,6 @@
 import useSeats from 'api/seats/useSeats';
 import { isNil } from 'utils/helperUtils';
-import { useAvailableSeats } from './useAvailableSeats';
+import useTotalSeats from './useTotalSeats';
 
 interface ReturnFunctionParameters {
```

Comment:
One last one. We could also change this to a descriptive name for when we want to use it elsewhere and also vscode ts reference
![image](https://user-images.githubusercontent.com/12659000/233126555-9e9eb031-6866-40c0-ba96-1654c82daaac.png)